matrix-meetings-bot/src/client/MeetingClient.test.ts

```diff
@@ -131,6 +131,101 @@ describe('MeetingClient', () => {
       },
     );
 
+    it('should create a meeting when default room version is 12', async () => {
+      const client = new MeetingClient(
+        new MatrixClient('', ''),
+        new EventContentRenderer({} as any),
+      );
+
+      await expect(
+        client.createMeeting(
+          undefined,
+          '@bot-user-id:example',
+          {
+            title: 'My Meeting',
+            description: 'My Description',
+            calendar: [
+              {
+                uid: 'uid-0',
+                dtstart: { tzid: 'Europe/Berlin', value: '20200507T100000' },
+                dtend: { tzid: 'Europe/Berlin', value: '20200507T110000' },
+              },
+            ],
+          },
+          MeetingType.MEETING,
+          { via: ['example'] },
```

Comment:
For the sake of avoiding future code debt lets follow rfc2606 and use `example.com` as the server part:


```suggestion
          { via: ['example.org'] },
```